### ISSUE TITLE
Feedback for Xiaomi page

### ISSUE THREAD

**Issue by GKid94** [urbandroid-team/dont-kill-my-app#898]
Add a new MIUI and HyperOS section https://dontkillmyapp.com/xiaomi

**Comment by jddcef** [comment]
Yes, interested in technical differences with hyperos, or is it the same just with a different skin?
